src/pages/developers/chains/functionality.mdx

```diff
@@ -1,40 +1,72 @@
+| :--------------------------------- | :---- | :------ | :------ | :-- | :------- |
+| Deposit/withdraw native coin       | EVM   | ✅      | ✅      | ✅  | ✅       |
+| Deposit/withdraw fungible token    | EVM   | ✅      | ✅      | ✅  | ✅       |
+| Deposit and call (native coin)     | EVM   | ✅      | ✅      | ✅  | ✅       |
+| Deposit and call (fungible token)  | EVM   | ✅      | ✅      | ✅  | ✅       |
+| Withdraw and call (native coin)    | EVM   | ✅      | ✅      | ✅  | ✅       |
+| Withdraw and call (fungible token) | EVM   | ✅      | ✅      | ✅  | ✅       |
+| Deposit ZETA                       | EVM   | ✅      | ✅      | ✅  | ❌       |
+| TSS Direct Deposits                | EVM   | ⚠️      | ⚠️      | ⚠️  | ❌       |
+| Withdraw ZETA                      | EVM   | ✅      | ✅      | ✅  | ❌       |
+| Deposit and call (ZETA)            | EVM   | ❌      | ❌      | ❌  | ❌       |
+| Withdraw and call (ZETA)           | EVM   | ❌      | ❌      | ❌  | ❌       |
+| Call                               | EVM   | ✅      | ✅      | ✅  | ✅       |
+
+## Bitcoin
+
+| Feature                        | Chain   | Mainnet | Testnet | E2E | Localnet |
+| :----------------------------- | :------ | :------ | :------ | :-- | :------- |
+| Deposit/withdraw native coin   | Bitcoin | ✅      | ✅      | ✅  | ❌       |
+| Deposit and call (native coin) | Bitcoin | ✅      | ✅      | ✅  | ❌       |
+
+## Solana
+
+| Feature                            | Chain  | Mainnet | Testnet | E2E | Localnet |
+| :--------------------------------- | :----- | :------ | :------ | :-- | :------- |
+| Deposit/withdraw native coin       | Solana | ✅      | ✅      | ✅  | ✅       |
+| Deposit/withdraw fungible token    | Solana | ✅      | ✅      | ❌  | ✅       |
+| Deposit and call (native coin)     | Solana | ✅      | ✅      | ✅  | ✅       |
+| Deposit and call (fungible token)  | Solana | ✅      | ✅      | ❌  | ✅       |
+| Withdraw and call (fungible token) | Solana | ❌      | ✅      | ✅  | ✅       |
+| Withdraw and call (native coin)    | Solana | ❌      | ✅      | ✅  | ✅       |
+| Deposit ZETA                       | Solana | ❌      | ❌      | ❌  | ❌       |
+| Withdraw ZETA                      | Solana | ❌      | ❌      | ❌  | ❌       |
+| Deposit and call (ZETA)            | Solana | ❌      | ❌      | ❌  | ❌       |
+| Withdraw and call (ZETA)           | Solana | ❌      | ❌      | ❌  | ❌       |
+| Call                               | Solana | ❌      | ❌      | ❌  | ❌       |
+
+## Sui
+
+| Feature                            | Chain | Mainnet | Testnet | E2E | Localnet |
+| :--------------------------------- | :---- | :------ | :------ | :-- | :------- |
+| Deposit/withdraw native coin       | Sui   | ❌      | ✅      | ✅  | ✅       |
+| Deposit/withdraw fungible token    | Sui   | ❌      | ❌      | ❌  | ❌       |
+| Deposit and call (native coin)     | Sui   | ❌      | ✅      | ✅  | ✅       |
+| Deposit and call (fungible token)  | Sui   | ❌      | ❌      | ❌  | ❌       |
```

Comment:
Fungible tokens supported for these, it was integrated along with SUI support

Reply:
Oh, we need to add support for Sui fungible tokens on localnet then.

Reply:
deb4306